exercises/queen-attack/src/main/java/Coordinate.java

```diff
@@ -0,0 +1,5 @@
+public final class Coordinate {
```

Comment:
Ah, yes, I did mean to leave a `BoardCoordinate` class in here since that's a required part of the exercise API; renamed!